tailwind.config.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
         'skin-block-bg': 'var(--block-bg)',
         'skin-header-bg': 'var(--header-bg)',
         'skin-heading': 'var(--heading-color)',
-        green: '#21b66f',
-        red: '#ff3856'
+        green: 'hsl(var(--green) / <alpha-value>)',
```

Comment:
What is the need for this change?

Reply:
This green is exactly the same as the hex value `#21b66f`
Making this change though, means you can add an alpha value to the color. for example adding the class `bg-green/50` would be `background-color: hsla(151 69% 42%, 0.5)`.

If the alpha value is omitted, like `bg-green` then an alpha value of 1 is assumed, so anywhere in the app you're referencing this green will still be the same.

We needed a way to stick to using your style guide's green but have some opacity variations, hope that makes sense!

Reply:
This is already working for green and red without your changes, e.g `bg-green/20` for 20% opacity on green background